cmd/harmony/main.go

```diff
@@ -496,6 +498,22 @@ func main() {
 		go currentNode.SupportBeaconSyncing()
 	}
 
+	////// Code only used for one-off rollback /////////
```

Comment:
Better print out a warning message in log/console so that we know the rollback happened, with more details.